s2s/dset/_eng2chi.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from typing import Sequence
+
+import pandas as pd
+from sklearn.metrics import accuracy_score
+
+from s2s.dset._base import BaseDset
+from s2s.path import DATA_PATH
+
+class Eng2ChiDset(BaseDset):
```

Comment:
Add docstring for `Eng2ChiDset` class, including reference of the source of the dataset.